### PR TITLE
fixing more places that used the old-style types db

### PR DESCRIPTION
**Pull request by mhrivnak**

### CODE REVIEW

File: server/pulp/server/managers/content/query.py [hunk 1]
```diff
@@ -22,11 +22,6 @@ def list_content_types(self):
         """
         return content_types_db.all_type_ids()
 
-    def get_content_type(self, type_id):
```
Comment (mhrivnak) [inline server/pulp/server/managers/content/query.py]: this isn't used anywhere, isn't documented, and obviously has no value based on the code you see below.

Comment (mhrivnak) [inline server/pulp/server/managers/content/query.py]: oh, and of course it also wasn't tested.

Comment (bmbouter) [inline server/pulp/server/managers/content/query.py]: :+1:

**Comment by bmbouter** [comment]
@mhrivnak LGTM! Good fix.

**Pull request merged**